### PR TITLE
[SMALLFIX] Fix backwards compatibility issue introduced in PR 3063.

### PR DESCRIPTION
**Pull request by jsimsa**
This PR fixes a backwards compatibility issue introduced by #3063

### CODE REVIEW

**Comment by yupeng9** [comment]
LGTM

File: core/common/src/main/java/alluxio/util/CommonUtils.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 @ThreadSafe
 public final class CommonUtils {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  static final String ALPHANUM = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/CommonUtils.java]: private? since no one is using it.

Comment (jsimsa) [inline core/common/src/main/java/alluxio/util/CommonUtils.java]: done

File: core/common/src/thrift/file_system_master.thrift [hunk 2]
```diff
@@ -36,7 +36,8 @@ struct ListStatusTOptions {
 struct FileBlockInfo {
   1: common.BlockInfo blockInfo
   2: i64 offset
-  3: list<string> ufsLocations
+  3: list<common.WorkerNetAddress> ufsLocations // deprecated since 1.1 will be removed in 2.0
```
Comment (apc999) [inline core/common/src/thrift/file_system_master.thrift]: `deprecated by ufsLocations`?

**Comment by apc999** [comment]
one minor comment left, otherwise LGTM

File: core/common/src/thrift/file_system_master.thrift (hunk 2 above)
Comment (jsimsa) [inline core/common/src/thrift/file_system_master.thrift]: you mean `ufsStringLocations`?

Comment (jsimsa) [inline core/common/src/thrift/file_system_master.thrift]: done

**Comment by apc999** [comment]
LGTM

**Pull request merged**